src/content/docs/browser/browser-integrations/mexn-integration.mdx

```diff
@@ -0,0 +1,95 @@
+
+### Step 2: Install MongoDB quickstart
+
+1. Check out our MongoDB agent requirements in our [documentation](/docs/infrastructure/host-integrations/host-integrations-list/mongodb-monitoring-integration/)  before installing the agent.
+2. Open the [MongoDB quickstart installation](https://newrelic.com/instant-observability/mongodb). 
+3. Click  **install now** at the bottom of the page to start the MongoDB agent installation.
+
+### Step 3: Install Node.js agent
+
+1. Check out our Node.js agent requirements in our [documentation](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).
+2. Open the [Node.js quickstart installation](https://newrelic.com/instant-observability/node-js). 
+3. Click  **install now** at the bottom of the page to start the Node.js agent installation.
+
+### Step 4: Install the browser agent [#browser-install] 
+
+To install our browser agent: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click **Browser and mobile**, then click **Browser monitoring**.
+3. Select the **Select copy/paste JavaScript code** option (for more about this code, see [our browser install docs](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent)). 
+4. Complete the rest of the steps and click **Enable.**
+5. Go the `index.html ` file of your React application or Angular application.
+6.  Paste our browser code snippet script tag right before the body tag ends (`</body>`).
+8. Restart your application. 
+
+In couple of minutes, your application will send metrics to [one.newrelic.com](https://one.newrelic.com).
+
+## Find your data [#find-data]
+
+To get your MExN dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click on **Dashboards**.
+3. In the search bar, type `MExN`. 
+4. The MExN dashboard should appear. Click on it to install it.
+
+Your MExN dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+For information about data reported, see the docs for each of the tools we had you install:
+
+* [Browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). 
+*  [Infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
+* [MongoDB data](docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-monitoring-integration-new/#find-and-use/).
+* [Node.js data](/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs/).
+
+
+Here's an example NRQL query checking the average window load time for the various browsers used by users on desktop computers: 
```

Comment:
This also sounds weird to me: "Here's an example NRQL query"
maybe these options?
Here's a NRQL query example checking...
Here's an example of a NRQL query

Reply:
👍